irr/include/CMeshBuffer.h

```diff
@@ -134,7 +134,7 @@ class CMeshBuffer final : public IMeshBuffer
 	//! Index buffer
 	SIndexBuffer *Indices;
 	//! Bounding box of this meshbuffer.
-	core::aabbox3d<f32> BoundingBox;
+	core::aabbox3d<f32> BoundingBox{{0, 0, 0}};
```

Comment:
Will this call the constructor `constexpr aabbox3d(const vector3d<T> &init) :` or what exactly will be initialized here? 
`vector3d<T> MinEdge;` and `vector3d<T> MaxEdge;` are already zero-initialized, thus this notation seems cumbersome.

Reply:
it will.

> thus this notation seems cumbersome.

wasn't my idea but it doesn't seem that unreasonable because it's easy to get things wrong with a zero-box

Reply:
Ideally the goal is to get rid of the default constructor in my opinion, because it misleads people into thinking they are getting an "empty" (in the set sense) aabb you can just add points to to get a correct aabb when there is no such thing.